### PR TITLE
Changed styling for search and main menu

### PR DESCRIPTION
**Pull request by martinyde**
https://leantime.itkdev.dk/#/tickets/showTicket/131

- Change search
- Change main menu

Menu:
billede

Search results:
billede

### CODE REVIEW

File: web/themes/custom/hoeringsportal/assets/css/module/_search.scss [hunk 1]
```diff
@@ -106,14 +110,20 @@
   }
 }
 
-.ui-autocomplete {
+.ui-autocomplete.ui-widget.ui-widget-content {
+  //display: block!important;
```
Comment (yepzdk) [inline web/themes/custom/hoeringsportal/assets/css/module/_search.scss:114]: Maybe this line can be deleted?

File: web/themes/custom/hoeringsportal/assets/css/module/_search.scss [hunk 2]
```diff
@@ -66,36 +70,36 @@
...
     display: block;
+    position: absolute;
+    top: 11px;
+    right: 3px;
   }
 
   &.is-open {
+    border: 1px solid rgba(133, 133, 133, .2);
+    background: $white;
+    padding-left: 1em;
```
Comment (yepzdk) [inline web/themes/custom/hoeringsportal/assets/css/module/_search.scss:90]: Maybe consider using the $spacer variable for paddings and margins. It equals 1rem.

**Review by yepzdk — COMMENTED** [review]
Looks good. A couple of minor comments in the code.

I regards to the design from the screenshots i have a couple of comments.

Menu:
* It looks like the search icon hangs a bit low, and do not properly align with the menu items.

Search:
* It looks like a border around the white background and not a shadow? 
* White background is also missing rounded corners.
* Rounded corners is missing on the focused result.
* The padding in the focused result seems a bit small, can we adjust 
* Can we limit the excerpt text to 2 lines?
* Is it possible to highlight the part of the text that matches the query text, as en the design?
* The spacing between the title and excerpt seems a bit too large.

File: web/themes/custom/hoeringsportal/assets/css/module/_search.scss (hunk 1 above)
Comment (martinyde) [reply web/themes/custom/hoeringsportal/assets/css/module/_search.scss]: Deleted

File: web/themes/custom/hoeringsportal/assets/css/module/_search.scss (hunk 2 above)
Comment (martinyde) [reply web/themes/custom/hoeringsportal/assets/css/module/_search.scss]: Changed.

**Comment by martinyde** [comment]
Changed screenshots in description after changes.

Menu:

    It looks like the search icon hangs a bit low, and do not properly align with the menu items.
    --- Moved a bit up

Search:

    It looks like a border around the white background and not a shadow?
    --- Added shadow on top of border as design i think.

    White background is also missing rounded corners.
    --- Added rounded corners

    Rounded corners is missing on the focused result.
    --- Added rounded corners

    The padding in the focused result seems a bit small, can we adjust
    --- Changed badding from .5em to .75em
    Can we limit the excerpt text to 2 lines?
    --- Reduced text length.

    Is it possible to highlight the part of the text that matches the query text, as en the design?
    --- Added highlighting in header and excerpt (if found in the first two lines of the excerpt). The result excerpt cannot easily be created from the search query so we can't center the text around the search query.

    The spacing between the title and excerpt seems a bit too large.
    --- Removed spacing

**Review by yepzdk — APPROVED** [review]

**Pull request merged**